platform-core-module/src/main/java/com/softicar/platform/core/module/file/smb/ISmbClient.java

```diff
@@ -1,43 +1,80 @@
 package com.softicar.platform.core.module.file.smb;
 
-import java.io.OutputStream;
-
 /**
  * Common interface of all SMB client implementations.
+ * <p>
+ * Provides functionality to access entries (files and folders) on an SMB share.
```

Comment:
on a SMB share

Reply:
In fact, phonetics matter here: SMB is pronounced "Ess-Emm-Bee". A preceding "a" hence becomes "an", because "Ess" (phonetic "S") starts with a vowel.